Lisp-Dep/fix-abcl.lisp

```diff
@@ -0,0 +1,16 @@
+(in-package :cl-user)
+
+(defpackage clim-mop
+  (:use cl mop)
+  (:import-from mop
+                compute-applicable-methods method-qualifiers class-name))
+
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (loop :for symbol :being :the :symbols :of :clim-mop
+     do (export symbol :clim-mop))
+    (loop :for symbol :being :the :external-symbols :of #:mop
+     do (export symbol #:mop)))
+  (loop :for symbol :being :the :external-symbols :of #:mop
+     do (export symbol #:mop)))
+
```

Comment:
we already use closer mop for that, so this fix is not necessary

Reply:
will fix